stories/Pagination/Pagination.story.jsx

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import styled from 'styled-components';
+import {
+  TabbedView,
+  Tab,
+  AutoPropsApi,
+  Heading,
+  StoryContainer,
+  Title,
+  SimpleHighlight,
+  Table,
+} from '@catho/quantum-storybook-ui';
+
+import { Pagination, Row, Col } from '../../components';
+import Mobile from '../../components/Pagination/sub-components/Mobile';
+import ControlledPagination from './examples/ControlledPagination';
+import ControlledPaginationAria from './examples/ControlledPaginationAria';
+
+const MobileWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+`;
+
+storiesOf('Pagination', module).add('Pagination', () => (
+  <>
+    <Heading name="Pagination">
+      Add pagination buttons to help split up your long content into shorter,
+      easier to understand blocks.
+    </Heading>
+    <TabbedView>
+      <Tab title="Usage">
+        <StoryContainer>
+          <Title as="h2">Importing Pagination</Title>
+          <SimpleHighlight>{`import { Pagination } from '@catho/quantum';`}</SimpleHighlight>
+
+          <p>
+            An adaptive Pagination component, if viewport width is below small
+            breakpoint value it will renders a mobile friendly component, if
+            not, will render an dotted Pagination component.
+          </p>
+
+          <p>
+            <strong>
+              It&apos;s highly recommended to avoid pagination and use infinite
+              scroll in small devices.
+            </strong>
+          </p>
+
+          <Title as="h2">Examples</Title>
```

Comment:
There is no need for this title, even their description. It is confusing because the description isn't describing the next example (Simple usage) and it is already mentioned [here](https://github.com/catho/quantum/pull/38/files#diff-65ccaff79a6419ef8cc47c36be2ea99bR209), I think.

It may start [here](https://github.com/catho/quantum/pull/38/files#diff-65ccaff79a6419ef8cc47c36be2ea99bR61) btw.

Reply:
Done.
Thanks!